web/crux-ui/locales/en/audit.json

```diff
@@ -1,3 +1,5 @@
 {
+  "token": "Token",
+  "deploymentTokenInitials": "dt",
```

Comment:
dt?

Reply:
fixed with an avatar as we discussed in person